CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This file is used to list changes made in each version of the aws-parallelcluste
 **ENHANCEMENTS**
 - Add possibility to override EC2 RunInstances parameters for instances launched in a Slurm cluster.
 
+**CHANGES**
+- Upgrade Slurm to version 21.08.3.
```

Comment:
What about: `Update Slurm plugin to support version 21.08`?

Reply:
Changed